orangecontrib/text/widgets/utils/tests/test_owbasevectorizer.py

```diff
@@ -0,0 +1,27 @@
+from orangecontrib.text import Corpus
+from orangecontrib.text.vectorization import BowVectorizer
+from orangecontrib.text.widgets.utils.owbasevectorizer import OWBaseVectorizer
+from orangewidget.tests.base import WidgetTest
+
+
+class TestableBaseVectWidget(OWBaseVectorizer):
+    name = "TBV"
+    Method = BowVectorizer
+
+
+class TestOWBaseVectorizer(WidgetTest):
+    def setUp(self):
+        self.widget = self.create_widget(TestableBaseVectWidget)
+        self.corpus = Corpus.from_file('deerwester')
+
+    def test_hide_attributes(self):
+        self.send_signal("Corpus", self.corpus)
+        self.assertTrue(all(f.attributes['hidden'] for f in
+                            self.get_output("Corpus").domain.attributes))
+        self.widget.controls.hidden_cb.setChecked(False)
+        self.assertFalse(all(f.attributes['hidden'] for f in
+                            self.get_output("Corpus").domain.attributes))
```

Comment:
`any` would fit better here -- in this case, it does not make difference but is more correct - just in case.

Reply:
Fixed.